diff --git a/src/term.c b/src/term.c
index c995a4499c..386238107f 100644
--- a/src/term.c
+++ b/src/term.c
@@ -4112,7 +4112,7 @@ init_tty (const char *name, const char *terminal_type, bool must_succeed)
   tty->TS_enter_alt_charset_mode = tgetstr ("as", address);
   tty->TS_exit_alt_charset_mode = tgetstr ("ae", address);
   tty->TS_exit_attribute_mode = tgetstr ("me", address);
-  tty->TS_enter_strike_through_mode = tgetstr ("smxx", address);
+  tty->TS_enter_strike_through_mode = tigetstr ("smxx");
 
   MultiUp (tty) = tgetstr ("UP", address);
   MultiDown (tty) = tgetstr ("DO", address);
