--- tty-keys.c
+++ tty-keys.c
@@ -957,25 +957,25 @@
 	 * Don't allow both KEYC_CTRL and as an implied modifier. Also convert
 	 * C-X into C-x and so on.
 	 */
-	if (nkey & KEYC_CTRL) {
-		onlykey = (nkey & KEYC_MASK_KEY);
-		if (onlykey < 32 &&
-		    onlykey != 9 &&
-		    onlykey != 13 &&
-		    onlykey != 27)
-			/* nothing */;
-		else if (onlykey >= 97 && onlykey <= 122)
-			onlykey -= 96;
-		else if (onlykey >= 64 && onlykey <= 95)
-			onlykey -= 64;
-		else if (onlykey == 32)
-			onlykey = 0;
-		else if (onlykey == 63)
-			onlykey = 127;
-		else
-			onlykey |= KEYC_CTRL;
-		nkey = onlykey|((nkey & KEYC_MASK_MODIFIERS) & ~KEYC_CTRL);
-	}
+	/* if (nkey & KEYC_CTRL) { */
+	/* 	onlykey = (nkey & KEYC_MASK_KEY); */
+	/* 	if (onlykey < 32 && */
+	/* 	    onlykey != 9 && */
+	/* 	    onlykey != 13 && */
+	/* 	    onlykey != 27) */
+	/* 		/\* nothing *\/; */
+	/* 	else if (onlykey >= 97 && onlykey <= 122) */
+	/* 		onlykey -= 96; */
+	/* 	else if (onlykey >= 64 && onlykey <= 95) */
+	/* 		onlykey -= 64; */
+	/* 	else if (onlykey == 32) */
+	/* 		onlykey = 0; */
+	/* 	else if (onlykey == 63) */
+	/* 		onlykey = 127; */
+	/* 	else */
+	/* 		onlykey |= KEYC_CTRL; */
+	/* 	nkey = onlykey|((nkey & KEYC_MASK_MODIFIERS) & ~KEYC_CTRL); */
+	/* } */
 
 	if (log_get_level() != 0) {
 		log_debug("%s: extended key %.*s is %llx (%s)", c->name,
